ds_judgements_public_ui/templates/includes/facets/court_search_facets.html

```diff
@@ -0,0 +1,10 @@
+{% load query_filters court_utils %}
+<div style="text-align: left;">
+  <b>Courts relevant to your query:</b>
+  {% for court, item_count in context.court_facets.items %}
+    {% if court.code not in context.query_params.court %}
+      <a href="{% url "search" %}?{{ request.GET.urlencode }}&court={{ court.code }}">{{ court.grouped_name }} ({{ item_count }})</a>
```

Comment:
This feels clunky and we should probably make it tidier in the real thing but OK for now

Reply:
Fixed! 😅